src/core/ext/filters/client_channel/health/health_check_client.cc

```diff
@@ -354,7 +354,8 @@ void HealthCheckClient::CallState::StartCall() {
   grpc_metadata_batch_init(&send_initial_metadata_);
   error = grpc_metadata_batch_add_head(
       &send_initial_metadata_, &path_metadata_storage_,
-      grpc_mdelem_from_slices(
+      // TODO(arjunroy): They're both static.
```

Comment:
What does this mean?

Reply:
The fastpath I have right now assumes a static first parameter and an interned second parameter - a fairly common occurrence. In this case both inputs are static so I could create a fastpath that assumes even more - but I have not done so thus far. So I'm just using the fastpath I've already created (which does a couple more ops than necessary for this case but is correct).

Reply:
Could you update your comment to reflect that? Something like: 
"// TODO(arjunroy): Both parameters are static so we could use that knowledge to create an even faster fastpath here." This is so that a general reader of this code can understand what this comment means.

Reply:
Added an overload for this case, so this comment no longer applies.